src/main/java/pl/sknikod/kodemy/util/NameCrudOperations.java

```diff
@@ -0,0 +1,15 @@
+package pl.sknikod.kodemy.util;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+public interface NameCrudOperations<RES,REQ> extends BaseCrudOperations<RES,REQ> {
```

Comment:
Nazwij to może `NameExtendCrudOperations`.
Bo teraz nazwa mówi jakby zmieniała `Id` na `Name`.